rfcs/0000-community-coordination-hub.md

```diff
@@ -0,0 +1,151 @@
+repository tooling.
+
+To implement, a detailed review of https://github.com/kubernetes/community
+shall be conducted as joint effort between the interested parties and as 
+an Annex I to this RFC.
+
+Based on this review, a clean repository shall be enacted implementing
+such aspects considered worth porting to our community.
+
+As a general guideline for this review, the vast success and impressive
+dynamics of the kubernetes ecosystem shall be acknowledged not only as a
+proof of concept, but as a very sucessfull and inspiring [viable system].
+
+[viable system]: https://en.wikipedia.org/wiki/Viable_system_theory
+
+# Brainstorm
+[brainstorm]: #brainstorm
+
+This section exposes loose ideas that have sparked interest of commenters
+in their order of submission. They shall be considered _optional_ material
+during implementation, whereas implementors shall have the liberty to
+deliver a consistent experience as they see fit.
+
+- [Community Event Calender](https://github.com/kubernetes/community/tree/master/events)
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+## As a newcomer
+As a newcomer, I'd be able to quickly find my way around, and start beeing
+useful before beeing turned down by inconsistent and spread-out onboarding.
+
+## As a veteran
+As a veteran, I'd be able to promote some ideas, that I care about,
+towards completion while suffering from less friction and overly complex
+coordination. I can leverage my experience and skills more effectively
+for the benefit of the community.
+
+## As a mentor
+As a mentor, I can _still_ \* maintain those high quality direct human
+interactions, while having full support of a _"curriculum-like"_ resource.
+
+\* _still_, as an aside, because nix community is known for it's great
+mentorship spirit.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+There _can_ be a better solution to the achieve _optimum levels of empowerment_
+for our community. We _could just **wait**_ for them to unveil themselves.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- Benevolent Dictatorship
+- Ephimeral Conversations (Discourse)
+
+The only current viable alternative for _coordination of resources and effort_
+are ephimeral conversations in Discourse. While, they lend meaning to the 
+individual's interpretations and views onto the ecosystem, they lack important
+features such as:
+
+- Structure
+- Consolidation
+- Ability to act as _Reference_
+
+A portal page might at first sight seem like an alternative, especially for those
+features. But, _optimum empowerment_ requires low barriers to interaction and
+amendment. Hence a git repository. Repository tooling can complement this aspect
+and deploy a static page under the nixos.org for a design lift on the content.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+Is this the right level of abstraction to act as an enabling RFC for future
```

Comment:
"enabling RFC"?

Reply:
a RFC that enables and triggers future improvement without bringing it about itself. Is there a better wording?

https://www.merriam-webster.com/dictionary/enabler

Reply:
Wasn't familiar with this word :).